scripts/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+#
+# build.sh runs a few commands to generate build time details
+# which are stored in variables in ./pkg/build
+
+commit=$(git rev-list -1 HEAD)
+version=$(git describe --tags)
+date=$(date -u +%Y-%m-%d-%H:%M:%S-%Z)
```

Comment:
I'm not sure we need this b/c it can be derived from version but ifso, we should use UTC time.

Reply:
Nevermind, didn't know -u eq --utc

Reply:
It doesn't, `%Z` inserts the time zone of the machine at build time. Should we convert to UTC?